README.md

```diff
@@ -0,0 +1,215 @@
+
+- `have installed ruby, git, npm, rails, gem`
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+  cd my-folder
+  git clone git@github.com:obibaadoma/Blog_App.git
+
+
+### Install
+
+Install this project with:
+
+  cd `Blog-App`
+  gem install rails
+  Add `rspec-rails` bundle in `gem` module
+  Bundle Install
+
+
+### Usage
+
+To run the project, execute the following command:
+
+  - Run the command `rails server`
+  - Start the rails server and append the corresponding `urls pattern`.
+
+### Test
+To test each unit tests of the project, execute the following command in your root directory of this repo.
+  - `rspec spec/folder/unit_test_spec.rb,` e.g rspec spec/controllers/users_controller.rb
+  - `rspec` - to test all the test units at a time.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Amare Kassa**
+
+- GitHub: [@githubhandle](https://github.com/obibaadoma)
+- Twitter: [@twitterhandle](https://twitter.com/obibakwekuadoma)
+- LinkedIn: [@linkedInhandle](https://www.linkedin.com/in/derek-akrasi-konadu/)
+
+👤 **EvanSnave**
+
+- GitHub: [EvansSnave](https://github.com/EvansSnave)
+- Twitter: [Kroja13](https://twitter.com/Kroja13)
+- LinkedIn: [Kevin Cruz](https://www.linkedin.com/in/kevin-cruz-25159a201/)
+
+👤 **Fisayo Michael Animashaun**
+
+- GitHub: [@fmanimashaun](https://github.com/fmanimashaun)
+- Twitter: [@fmanimashaun](https://twitter.com/fmanimashaun)
+- LinkedIn: [Fisayo Michael Animashaun ](https://linkedin.com/in/fmanimashaun)
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- **Authorizing logged in users**
+- **Creating API for the application**
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+> Feel free to check the [issues page](https://github.com/obibaadoma/Blog_App/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+> Please ⭐️ this repository if you think the project is useful for you.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+I would like to thank ruby on rails guide which is full of rich information and again Microverse that provides the right material that helps me to improve skills on RoRs.
+
+
+## ❓ FAQ <a name="faq"></a>
+
+
+- **Does this project milestone accept data via forms?**
+  - Yes
+
+- **Do you see the right information in each template for data you inserted via `rails c`**
+
+  - Yes
+
+   **Did you additional fields other than the model field in the form**
+
+  - No
+
+   **Does the optimized query improve the preformance of the query speed?**
+
+  - Yes
+
+  **did you implement the authentication from the scratch**
+
+  - No
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed...
```

Comment:
- [ ] Please fix the `LICENSE` broken link issues, because your `LICENSE` file doesn't exist, seems like you are license file name is the `MIT.md` file but you are passing the `path` for the `LICENSE` file. Kindly fix the issue.

![image](https://github.com/egichuhis/my-rails-blog/assets/66476329/f9f7bbeb-2cbc-4085-97d6-92290387cd88)

Reply:
added license file